superset/models/core.py

```diff
@@ -492,6 +492,8 @@ def _get_sqla_engine(
                     source = utils.QuerySource.DASHBOARD
                 elif "/explore/" in request.referrer:
                     source = utils.QuerySource.CHART
+                elif "/sqllab/" in request.referrer:
+                    source = utils.QuerySource.SQL_LAB
                 elif "/superset/sqllab" in request.referrer:
```

Comment:
Do we need to preserve the old `/superset/sqllab`?

Reply:
Good catch